### PR TITLE
Add KB article on organizing apps with workspaces

### PR DESCRIPTION
**Pull request by snehankekre**
This PR adds @carolinedlu's  KB article on organizing apps with workspaces on Streamlit Cloud.

### CODE REVIEW

File: content/kb/deployments/organize-apps-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Personal workspace
+
+If an app's GitHub repository is owned by you, the app will appear in your personal workspace, named "<YourGitHubHandle\>".
+
+![App workspaces 2](/images/streamlit-cloud/app-workspaces-2.png)
+
+## Organization workspace
+
+If an app's GitHub repository is owned by **an organization** (such as your company), the app will appear in a separate workspace, named "<GitHubOrganizationHandle\>".
```
Comment (randyzwitch) [inline content/kb/deployments/organize-apps-workspaces.md:20]: There's an errant space in the quotation mark around githuborghandle, which causes a weird line break

**Review by randyzwitch — APPROVED** [review]
Approved pending fixing the space

File: content/kb/deployments/organize-apps-workspaces.md (hunk 1 above)
Comment (snehankekre) [reply content/kb/deployments/organize-apps-workspaces.md:20]: @randyzwitch I neither see the errant space nor the weird line break :thinking: . Would you mind sharing a screenshot of the line break?

Comment (snehankekre) [reply content/kb/deployments/organize-apps-workspaces.md:20]: I see the line break in the GitHub UI, but not when inspecting the file content locally or on the rendered page. Not sure how to proceed :sweat_smile:

Comment (randyzwitch) [reply content/kb/deployments/organize-apps-workspaces.md:20]: That is so weird, it's gone now...that's an odd thing for me to have imagined, yet, no commits have made between now and then.

Ready for merge :exploding_head:

**Pull request merged**